### PR TITLE
docs(Tasklist API): Add note on JWT token usage

### PR DESCRIPTION
**Pull request by johanwelgemoed**
## What is the purpose of the change

Add note to ClaimTask section clarifying the usage of JWT token

## Are there related marketing activities
N/A

## PR Checklist

- [x] My changes apply to an already released version, and I have added them to the relevant `/versioned_docs` directory, or they are not for an already released version.
- [x] My changes apply to future versions, and I have added them to the main `/docs` directory, or they are not for future versions.
- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

File: docs/apis-clients/tasklist-api/mutations/claim-task.mdx [hunk 1]
```diff
@@ -5,6 +5,14 @@ title: claimTask
 
 Claim a task with taskId to currently logged in user. Returns the task.
 
+:::note
```
Comment (christinaausley) [inline docs/apis-clients/tasklist-api/mutations/claim-task.mdx:8]: Just a quick technical note -- you'll need to adjust "m2m" to "M2M" and "therefor" to "therefore" in these notes 👍

Comment (christinaausley) [reply docs/apis-clients/tasklist-api/mutations/claim-task.mdx:8]: I adjusted this in my latest commit, but let me know if you have any questions.

**Review by sdorokhova — CHANGES_REQUESTED** [review]
This part of documentation is auto-generated from our [GraphQL schema file](https://github.com/camunda/tasklist/blob/master/webapp/src/main/resources/graphql/tasklist.graphqls) 
Therefore we should not manually adjust `claim-task.mdx`. We can leave the adjustments of older versions, but for the future version, we need to add a note to schema file in Tasklist repository.

File: docs/apis-clients/tasklist-api/mutations/claim-task.mdx [hunk 2]
```diff
@@ -3,15 +3,7 @@ id: claim-task
 title: claimTask
 ---
 
-Claim a task with taskId to currently logged in user. Returns the task.
```
Comment (johanwelgemoed) [inline docs/apis-clients/tasklist-api/mutations/claim-task.mdx:6]: Removed change here as it needs to be done via a schema change as a separate PR for next versions.

File: docs/apis-clients/tasklist-api/mutations/claim-task.mdx [hunk 3]
```diff
@@ -3,7 +3,7 @@ id: claim-task
...
+Claim a task with taskId to currently logged in user. Returns the task.ß
```
Comment (christinaausley) [inline docs/apis-clients/tasklist-api/mutations/claim-task.mdx:6]: What does this addition symbolize? I may need to add to the style guide.

**Review by sdorokhova — APPROVED** [review]

File: docs/apis-clients/tasklist-api/mutations/claim-task.mdx (hunk 3 above)
Comment (johanwelgemoed) [reply docs/apis-clients/tasklist-api/mutations/claim-task.mdx:6]: Good catch. I think that was a weird leftover symbol when I removed the text. Not sure whre it came from or why it appeared. I Removed it.

Comment (christinaausley) [reply docs/apis-clients/tasklist-api/mutations/claim-task.mdx]: I was thinking "hm, I don't know that I've seen this in technical writing before but maybe it's common knowledge for beta or something?" 😆

**Review by christinaausley — APPROVED** [review]
